.github/workflows/release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY_URL: docker.io
+  REGISTRY_NAMESPACE: mrutski
+
+jobs:
+
+  build:
+    runs-on: ubuntu-20.04
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    services:
```

Comment:
For some reason the maven plugin `io.fabric8:docker-maven-plugin` which runs the following containers does not work within github worker. This declaration does that instead, and maven plugin execution is disabled via `-Ddocker.skip=true` flag